drivers/quota/projectquota.go

```diff
@@ -129,7 +151,7 @@ func NewControl(basePath string) (*Control, error) {
 
 	q := Control{
 		backingFsBlockDev: backingFsBlockDev,
-		nextProjectID:     minProjectID + 1,
+		nextProjectID:     minProjectID,
```

Comment:
`minProjectID` was already used for something just above. Why is it OK to reuse it here? It might well be, but the commit message / PR description doesn’t say that it is, or even that it is changing this.

Reply:
Ah, the `setProjectQuota` above seems to be a no-op.

Reply:
I reworked it and if the user configures the system correctly then the overflow IDs will no longer be a problem. Less hacky.